src/addandremove.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
+
+import {
+  //   clearAll,
+  form,
+  arr,
+  editFormArr,
+} from './index.js';
+
+export function addTodo() {
+  form.addEventListener('submit', () => {
+    const input = document.querySelector('.addlist');
+
+    const toDo = input.value;
+    if (toDo) {
+      arr.push({
+        name: toDo,
+        completed: false,
+        index: arr.length + 1,
+      });
+
+      localStorage.setItem('List', JSON.stringify(arr));
+    }
+    input.value = '';
+  });
+}
+
+export function editTask() {
+  editFormArr.forEach((form) => {
+    form.addEventListener('submit', (e) => {
+      e.preventDefault();
+      const { value } = form.querySelector('.text');
+      const id = Number(form.querySelector('.text').id);
+      const newArr = JSON.parse(localStorage.getItem('List')) || [];
+      newArr.forEach((item) => {
+        if (item.index === id) {
+          item.name = value;
+          localStorage.setItem('List', JSON.stringify(newArr));
+          location.reload();
+        }
+      });
+    });
+  });
+}
+
+export function removeElement(task, trash) {
+  const tasks = Array.from(task);
+  tasks.forEach((task) => {
+    task.addEventListener('click', () => {
+      tasks.forEach((task) => {
+        task.classList.remove('focus');
+      });
+      task.classList.add('focus');
+    });
+  });
```

Comment:
Great job @dieum512 🤝

- [ ] As stated in the project requirements, deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1).
![delete](https://user-images.githubusercontent.com/63917725/222198131-e4560914-2b96-4f99-b762-7f4289227577.PNG)